frontend/src/pages/EnterReceiptPage/__tests__/enterReceiptPage.spec.tsx

```diff
@@ -85,6 +85,43 @@ describe("EnterReceiptPage Component", () => {
     });
   });
 
+  it("prevents duplicate sdn in table when selecting search for sdn", async () => {
+    const { getByPlaceholderText, getByText, container } = render(
+      <EnterReceiptPage />,
+      {
+        authContext: {
+          apiCall: fetchMock
+        }
+      }
+    );
+    const inputField = getByPlaceholderText("SDN");
+    const submit = getByText("Search");
+    fireEvent.change(inputField, {
+      target: { value: "samesdn" }
+    });
+    await wait(() => {
+      expect(inputField).toHaveValue("samesdn");
+    });
+    fireEvent.click(submit);
+    await wait(() => {
+      expect(getByText("samesdn")).toBeInTheDocument();
+    });
+    fireEvent.change(inputField, {
+      target: { value: "samesdn" }
+    });
+    await wait(() => {
+      expect(inputField).toHaveValue("samesdn");
+    });
+    fireEvent.click(submit);
+    await wait(() => {
+      expect(getByText("samesdn")).toBeInTheDocument();
+      const errorMsg = container.querySelector(
+        "div.usa-alert--error"
+      ) as Element;
+      expect(errorMsg).toBeInTheDocument();
```

Comment:
Can you check that the text, "SDN already exists in table" is in the document in addition to or instead of checking for the error class?

Reply:
Changed to expect text for simplicity